index.html

```diff
@@ -70,5 +70,220 @@ <h3>LET'S CONNECT</h3>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>Back End Dev</p>
+					</li>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>2015</p>
+					</li>
+				</ul>
+				<p>
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<ul class="works__card--tags">
+					<li>HTML</li>
+					<li>CSS</li>
+					<li>JavaScript</li>
+				</ul>
+				<button type="button">See project</button>
+			</div>
+			<div class="works__card">
+				<img src="assets/img/Project3.svg" alt="project three" />
+				<h2>Tonic</h2>
+				<ul class="works__card--info">
+					<li><span>CANOPY</span></li>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>Back End Dev</p>
+					</li>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>2015</p>
+					</li>
+				</ul>
+				<p>
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<ul class="works__card--tags">
+					<li>HTML</li>
+					<li>CSS</li>
+					<li>JavaScript</li>
+				</ul>
+				<button type="button">See project</button>
+			</div>
+			<div class="works__card">
+				<img src="assets/img/Project4.svg" alt="project four" />
+				<h2>Multi-Post Stories</h2>
+				<ul class="works__card--info">
+					<li><span>CANOPY</span></li>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>Back End Dev</p>
+					</li>
+					<li>
+						<img src="assets/img/Counter.svg" alt="counter" />
+						<p>2015</p>
+					</li>
+				</ul>
+				<p>
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<ul class="works__card--tags">
+					<li>HTML</li>
+					<li>CSS</li>
+					<li>JavaScript</li>
+				</ul>
+				<button type="button">See project</button>
+			</div>
+		</section>
+
+		<section class="about" id="about">
+			<div class="about__main-content">
+				<h2>About Myself</h2>
+				<p>
+					I’m a software developer! I can help you build a product , feature or
+					website Look through some of my work and experience! If you like what
+					you see and have a project you need coded, don’t hesitate to contact
+					me.
+				</p>
+				<h3>LET'S CONNECT</h3>
+				<ul class="about__main-content--social">
+					<li>
+						<a href="#">
+							<img src="assets/img/Github.svg" alt="github" />
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<img src="assets/img/Linkedin.svg" alt="linkedin" />
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<img src="assets/img/AngelList.svg" alt="angel list" />
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<img src="assets/img/Twitter.svg" alt="twitter" />
+						</a>
+					</li>
+					<li>
+						<a href="#">
+							<img src="assets/img/Medium.svg" alt="medium" />
+						</a>
+					</li>
+				</ul>
+				<button type="button">Get my resume</button>
```

Comment:
- [ ] great work so far. Please kindly implement the styling: Active, Hover and Pressed on your button so it matches the figma design


![Screenshot (896)](https://user-images.githubusercontent.com/65298048/227598307-f37efbaa-557c-47c4-a90f-6ae80d3de383.png)